server/coordinator/node_picker.go

```diff
@@ -6,11 +6,16 @@ import (
 	"context"
 	"crypto/rand"
 	"math/big"
+	"time"
 
 	"github.com/CeresDB/ceresmeta/server/cluster/metadata"
 	"github.com/pkg/errors"
 )
 
+const (
+	expiredThreshold = time.Second * 10
```

Comment:
I guess whether a node is expired should be decided in the `node_picker`. Maybe you should move this threshold to the metadata package.